.github/workflows/llvm-project.yml

```diff
@@ -0,0 +1,31 @@
+name: Build kernel.org toolchain
+
+on: workflow_dispatch
+jobs:
+  stage_1:
+    name: stage 1
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        image: [llvm-project]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to GitHub Container Registry
```

Comment:
Add the same logic below for this step so that it only logs into the container registry if we are going to push an image:

```
- name: ...
  if: ${{ github.repository_owner == 'ClangBuiltLinux' && github.event_name != 'pull_request' && github.ref_name == 'main' }}
```

This obviously does not matter now, since there is no way to run this workflow on a pull, but it will make running this in a more automated fashion later more reliable and safe.

Reply:
done PTAL